sysid_data20210213-1223.json

```diff
@@ -0,0 +1,5 @@
+{
+"test": "Drivetrain",
+"units": "Meters",
+"unitsPerRotation": 1.0
+}
```

Comment:
Is this supposed to be here?

Reply:
Nope, I'll remove it in the next commit.